src/raftstore/store/peer.rs

```diff
@@ -369,8 +369,39 @@ impl Peer {
         where T: Transport
     {
         for msg in msgs {
-            if msg.get_msg_type() == MessageType::MsgRequestVote {
-                PEER_RAFT_READY_COUNTER_VEC.with_label_values(&["request_vote"]).inc();
+            match msg.get_msg_type() {
```

Comment:
The metrics are moved to `Store` and flushed on every tick. The sysbench test with tidb shows that they do not impact the performance notably.